ckanext/dalrrd_emc_dcpr/plugin.py

```diff
@@ -15,3 +20,85 @@ def update_config(self, config_):
 
     def get_commands(self):
         return [test_ckan_cmd]
+
+    # IDatasetForm
+
+    def _admins_only_create(self, key, data, errors, context):
```

Comment:
Whenever you write a function, please take a moment to think about its name, as good naming is very helpful when the code is being read. For methods and functions, it is usually a good idea to use verbs in the name. This helps clarifying that this is a function and that it performs some action.

Instead of `_admins_only_create()`, perhaps use `check_for_admin` or something similar.